### PR TITLE
Add default description and capture errors

### PR DESCRIPTION
**Pull request by alanfriedman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "uxshot",
-  "version": "0.0.21",
+  "version": "0.0.22",
   "main": "dist/uxshot.js",
   "license": "MIT",
   "repository": "github:alanfriedman/uxshot",
   "scripts": {
+    "dev": "npx webpack -w",
```
Comment (maniator) [inline package.json:8]: yarn build -w :-)

File: src/index.js [hunk 2]
```diff
@@ -61,14 +86,33 @@ export function init({
...
+        defaultDescriptionData,
+        description
+      );
     }
 
-    // const res = await fetch("http://localhost:3000/upload",
-    const res = await fetch('https://uxshot.com/upload', {
+    formData.append('description', description);
+
+    formData.append('errors', JSON.stringify(jsErrors));
```
Comment (maniator) [inline src/index.js:113]: Clean up errors after pushing?

File: src/index.js [hunk 3]
```diff
@@ -3,12 +3,37 @@ import hotkeys from 'hotkeys-js';
...
+      },
+    });
+  });
+}
+
+function getDefaultDescriptionData() {
+  return {
+    userAgent: window.navigator.userAgent,
+    windowSize: `${window.innerWidth}x${window.innerHeight}`,
+    language: navigator.language,
```
Comment (maniator) [inline src/index.js]:
```suggestion
    language: window.navigator.language,
```

For consistency

File: src/index.js [hunk 4]
```diff
@@ -61,14 +86,33 @@ export function init({
...
-    let description;
+    const defaultDescriptionData = getDefaultDescriptionData();
+    const {
+      userAgent,
+      windowSize,
+      url: pageUrl,
+      language,
+    } = defaultDescriptionData;
+    let description = `
+      Time: ${new Date()}
```
Comment (maniator) [inline src/index.js:97]: `new` is not needed.

**Review by maniator — APPROVED** [review]
Consider comments.

File: src/index.js (hunk 4 above)
Comment (alanfriedman) [reply src/index.js:97]: Wait, really?

**Pull request merged**